compact/tracker.xml

```diff
@@ -119,12 +119,25 @@
       vis="TrackerSubAssemblyVis">
       <composite name="InnerMPGDBarrel"/>
     </detector>
+    <detector id="TrackerSubAssembly_4_ID"
+      name="BarrelTOFSubAssembly"
+      type="DD4hep_SubdetectorAssembly"
+      vis="TrackerSubAssemblyVis">
+      <composite name="BarrelTOF"/>
+    </detector>
     <detector id="TrackerSubAssembly_5_ID"
       name="OuterMPGDSubSubAssembly"
       type="DD4hep_SubdetectorAssembly"
       vis="TrackerSubAssemblyVis">
       <composite name="OuterMPGDBarrel"/>
     </detector>
+    <detector id="TrackerSubAssembly_6_ID"
+      name="EndcapTOFSubSubAssembly"
+      type="DD4hep_SubdetectorAssembly"
+      vis="TrackerSubAssemblyVis">
+      <composite name="ForwardTOF"/>
+      <composite name="BackwardTOF"/>
+    </detector>
```

Comment:
This has overlap with #191 where it is reported as failing by Nicolas. @sly2j was going to look into that, and should review here.

Reply:
Yes. The recommendation from Tuesday's CompSW meeting by Joe and Sylvester is to move forward with merging this into main branch, while investigating into #191.